Client/src/Pages/PageSpeed/Details/index.css

```diff
@@ -51,3 +51,18 @@
   width: 22px;
   height: 22px;
 }
+.page-speed-details .MuiPieArcLabel-root {
+  font-size: var(--env-var-font-size-small-plus);
+  transition: fill 300ms ease;
+}
+.page-speed-details .MuiPieArcLabel-faded {
+  fill: rgba(0, 0, 0, 0.3);
+}
+.page-speed-details .pie-label,
+.page-speed-details .pie-value-label {
+  transition: all 400ms ease;
```

Comment:
Can we slow down the animations a little bit?  Seems a bit jumpy to me when hovering over, might be nice to have the animation take a little longer :thinking:

Reply:
You're referring to the pie arc animations right? I haven't touched those so they're default as they come from @mui. I can possibly override them though. I'll look into it.